api/src/main/java/jakarta/data/metamodel/Attribute.java

```diff
@@ -69,13 +69,13 @@
  * }
  *
  * public enum Color { BLACK, BLUE, GRAY, RED, WHITE }
- * </pre>
+ * }</pre>
  *
  * <p>The static metamodel class (typically generated from the entity class)
  * would be,</p>
  *
- * <pre>
- * &#64;StaticMetamodel
+ * <pre>{@code
+ * @StaticMetamodel
```

Comment:
The entity class is required by the annotation,
```suggestion
 * @StaticMetamodel(Car.class)
```